src/components/util/InputReset.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { keen } from "style/theme";
+import { IconX } from "components/Glyphs";
+
+const InputReset = ({ visible = false, onClick }) => {
+  return (
+    <Button visible={visible} onClick={onClick}>
+      <IconX />
+    </Button>
+  );
+};
+
+const Button = styled.button`
```

Comment:
Seeing that button appear just as I'm entering text is a little distracting. How about fading it in with an opacity transition?

```suggestion
const Button = styled.button`
  visibility: ${props => (props.visible === true ? "visible" : "hidden")};
  opacity: ${props => (props.visible === true ? "1" : "0")};
```